console/billing/reference-content/reading-an-invoice.mdx

```diff
@@ -0,0 +1,32 @@
+---
+meta:
+  title: Reading an invoice
+  description: This page explains how to read an invoice
+content:
+  h1: Reading an invoice
+  paragraph: This page explains how to read an invoice
+tags: billing invoice payment 
+dates:
+  validation: 2024-02-23
+  posted: 2024-02-23
+categories:
+  - billing
+---
+
+<Lightbox src={["Facture1.webp", "Facture2.webp"]} size={"medium"} alt={["Facture1", "Facture2"]} />
+
+1. The invoice number. This is a unique reference number for your invoice.
+2. The status of the invoice showing the billing period, the invoice's issuing and due dates.
+3. The seller (Scaleway SAS) contact information and VAT number.
+4. The customer information, your postal address, e-mail address, Organization ID, and in case of a business account your EU VAT number.
```

Comment:
```suggestion
4. The customer information, your postal address, email address, Organization ID, and in case of a business account your EU VAT number.
```